internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -16,12 +16,18 @@ import (
 	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
 )
 
+type AMSQuotaService interface {
```

Comment:
I understand this interface is temporary and also the passing of it on the other types right?

Reply:
yes. It is just needed by the temporary reconciler to get the subscription. It will be removed together with the reconciler